back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -26,7 +26,7 @@
 
     # the name of this variable shouldn't be `folder`
     # because otherwise it will be used by default in `parameter :folder`
-    let!(:custom_folder) { create(:project_folder, projects: projects.take(3)) }
+    let!(:custom_folder) { create(:project_folder, projects: [projects[0..2]]) }
```

Comment:
Aren't both equivalent? If so, I personally find `.take(3)` to be more readable.

Reply:
OK, I have no strong opinion on this.